packages/react/src/globals/_charts.scss

```diff
@@ -1,5 +1,13 @@
-@import '~@carbon/charts/styles/styles.scss';
-@import './vars';
+@use 'sass:meta';
+@use '~@carbon/styles/scss/compat/themes' as compat;
+
+// backwards compat. until @carbon/charts updates their scss
+$carbon--theme--g100: compat.$g100;
+$carbon--theme--g90: compat.$g90;
+$carbon--theme--g10: compat.$g10;
+$carbon--theme: compat.$g10;
+
+@import '@carbon/charts/styles/styles';
```

Comment:
We still want `@import` here?

Reply:
yeah, carbon/charts doesn't support use yet and all the themes get lost when using `@use`